### PR TITLE
[chore] Gateway 2.8.1.3 support for Lambda plugin

### PR DESCRIPTION
**Pull request by Guaris**
### Summary
#4142

### CODE REVIEW

File: app/_hub/kong-inc/aws-lambda/_index.md [hunk 1]
```diff
@@ -238,11 +260,26 @@ argument to the AWS Lambda function.
...
+invoke the Lambda function.
+
 If you do not provide an `aws_key` and `aws_secret`, the plugin uses an IAM role inherited
 from the instance running Kong.
 
-First, the plugin tries ECS metadata to get the role. If no ECS metadata is available,
-the plugin falls back on EC2 metadata.
+For example, if you're running Kong on an EC2 instance, the IAM role that attached
+to the EC2 will be used, and Kong will fetch the credential from the
+[EC2 Instance Metadata service(IMDSv1)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html);
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/aws-lambda/_index.md:271]:
```suggestion
[EC2 Instance Metadata service(IMDSv1)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html).
```

File: app/_hub/kong-inc/aws-lambda/_index.md [hunk 2]
```diff
@@ -238,11 +260,26 @@ argument to the AWS Lambda function.
...
+ if you're running Kong in an ECS container, the task IAM role will be used, and Kong will fetch the credentials from
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/aws-lambda/_index.md:272]:
```suggestion
If you're running Kong in an ECS container, the task IAM role will be used, and Kong will fetch the credentials from
```

File: app/_hub/kong-inc/aws-lambda/_index.md [hunk 3]
```diff
@@ -238,11 +260,26 @@ argument to the AWS Lambda function.
...
+the [container credential provider](https://docs.aws.amazon.com/sdkref/latest/guide/feature-container-credentials.html).
+Note that the plugin will firstly try to fetch from ECS metadata to get the role, and if no ECS metadata related environment
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/aws-lambda/_index.md:274]:
```suggestion
Note that the plugin will first try to fetch from ECS metadata to get the role, and if no ECS metadata related environment
```

File: app/_hub/kong-inc/aws-lambda/_index.md [hunk 4]
```diff
@@ -238,11 +260,26 @@ argument to the AWS Lambda function.
...
+variables is available, the plugin falls back on EC2 metadata.
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/aws-lambda/_index.md:275]:
```suggestion
variables are available, the plugin falls back on EC2 metadata.
```

File: app/_hub/kong-inc/aws-lambda/_index.md [hunk 5]
```diff
@@ -238,11 +260,26 @@ argument to the AWS Lambda function.
...
+
+If you also provide the `aws_assume_role_arn` option, the plugin will try to perform
+an additional [AssumeRole](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)
+action(which requires the Kong process to make HTTPS request to AWS STS service API) after
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/aws-lambda/_index.md:279]:
```suggestion
action, which requires the Kong process to make HTTPS request to AWS STS service API, after
```

File: app/_hub/kong-inc/aws-lambda/_index.md [hunk 6]
```diff
@@ -238,11 +260,26 @@ argument to the AWS Lambda function.
...
+configuring AWS access key/secret or fetching credentials automatically from EC2/ECS IAM roles,
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/aws-lambda/_index.md:280]:
```suggestion
configuring AWS access key/secret or fetching credentials automatically from EC2/ECS IAM roles.
```

File: app/_hub/kong-inc/aws-lambda/_index.md [hunk 7]
```diff
@@ -238,11 +260,26 @@ argument to the AWS Lambda function.
...
+and if succeeded, the plugin will fetch a temporary security credentials which represents
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/aws-lambda/_index.md:281]:
```suggestion
If it succeeds, the plugin will fetch a temporary security credentials that represents
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]

**Review by cloudjumpercat — APPROVED** [review]

**Review by tyler-ball — APPROVED** [review]

**Comment by tyler-ball** [comment]
@Guaris Is this supposed to still be `do not review`?

**Pull request merged**